app/views/layouts/application.html.erb

```diff
@@ -22,6 +22,26 @@
     <%= render 'layouts/footer' %>
 
     <script>
+      $("input#search-input").on({
```

Comment:
I think this is better moved into a separate file rather than `application.html.erb`, to stick with modularity

Reply:
@sashadev-sky I also think But I don't find any file to add this,So I was thinking to add all script in new .js file.
thank you : )

Reply:
@sashadev-sky I also think But I don't find any file to add this,So I was thinking to add all script in new .js file.
thank you : )